tests/test_individual_metrics.py

```diff
@@ -0,0 +1,48 @@
+"""Ensure that the functions to count the lines
+
+of code program are correct.
+"""
+
+import pytest
+
+from src import individual_metrics
+from src import print_table
+
+
+# TODO fix test case
```

Comment:
TODOs should not be merged into master, nor should a broken test case.

Reply:
This PR is a work in progress as shown in the label above, and these test cases will be fixed before merging. The issues with these test cases go beyond just the modules that we created and specifically have to do with the architects' team which you are a part of. I have already raised the necessary issue and I am waiting for their response.